tests/test_neo4j.py

```diff
@@ -1,57 +1,103 @@
+from __future__ import print_function
```

Comment:
Can we rename the filename back to `test_login.py`?

Hopefully get around to migrating everything to neo4j... in which case this filename may be slightly inappropriate...

Reply:
Done, thanks.